src/color-picker/color-picker.scss

```diff
@@ -235,9 +240,10 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb
 }
 
 @mixin iui-color-dot {
+  --iui-color-dot-inset: 0 0;
```

Comment:
sure its `0 0`? in html you defined with auto right and bottom values

Reply:
it works (dot appears at top left) but still I changed to `initial`

Reply:
with `initial`, it just uses the default value, which is `auto` so it will be fine. I tested and it's placing the dot at the top left